pkg/snapshot/velero.go

```diff
@@ -0,0 +1,74 @@
+package snapshot
+
+import (
+	"github.com/replicatedhq/kots/pkg/kotsadm"
+
+	"context"
+
+	"github.com/pkg/errors"
+	veleroclientv1 "github.com/vmware-tanzu/velero/pkg/generated/clientset/versioned/typed/velero/v1"
+	kuberneteserrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+func CreateRBACPermissions(kotsadmNamespace string) error {
+	veleroNamespace, err := DetectVeleroNamespace()
+	if err != nil {
+		return errors.Wrap(err, "failed to detect velero namespace")
+	}
+	if veleroNamespace == "" {
+		return errors.New("velero not found")
```

Comment:
How about logging this on INFO level instead of aborting.  Again, to not make this function depend on Velero's existence.

Reply:
i intentionally made this a sub cli command in the `backup` command and made it velero specific so that users won't abuse/misuse it.